### ISSUE TITLE
Don't ask me again in Google Translate doorhanger panel is missing its checkbox

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#14767]
## Description 
`Don't ask me again` in Google Translate doorhanger panel is missing its checkbox

## Steps to Reproduce

   1.  new profile on `nightly`
   2.  load `https://pgl.yoyo.org/adservers/serverlist.php?hostformat=adblockplus&showintro=1&mimetype=plaintext`
   3.  look at the doorhanger panel which drops down from the URL bar

## Actual result:

<img width="1312" alt="Screen Shot 2021-03-16 at 1 58 20 PM" src="https://user-images.githubusercontent.com/387249/111378879-aef15a00-865f-11eb-88d7-2b0d3d0b13e8.png">

## Expected result:

I'd expect there to be a checkbox to the left of the `Don't ask me again` text (compare with the Widevine prompt, below)

<img width="772" alt="Screen Shot 2021-03-16 at 1 58 52 PM" src="https://user-images.githubusercontent.com/387249/111379042-e4964300-865f-11eb-9076-9117fd092341.png">

And, like the Widevine prompt, it should also be:
1. keyboard-accessible (tab with focus rings, spacebar to select and deselect checkbox/option) - cross-reference: https://github.com/brave/brave-browser/issues/14578
2. text-consistent (`Don't ask me again` here, vs. `Don't ask again` in Widevine prompt)

## Reproduces how often: 
100%

## Brave version (brave://version info)

Brave | 1.23.41 Chromium: 89.0.4389.90 (Official Build) nightly (x86_64)
-- | --
Revision | 62eb262cdaae9ef819aadd778193781455ec7a49-refs/branch-heads/4389@{#1534}
OS | macOS Version 11.2.3 (Build 20D91)

cc: @karenkliu @rebron 

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the nightly channel? yes

**Comment by rebron** [comment]
cc: @yrliou When you get a chance can you take a look?

**Comment by GKid94** [comment]
Missing in right click menu.

<img width="1057" height="575" alt="Image" src="https://github.com/user-attachments/assets/c4aed57b-6635-486a-841e-82e2cacd4e6c" />
